aspnet/webhooks/receiving/receivers.md

```diff
@@ -28,7 +28,7 @@ https://<host>/api/webhooks/incoming/<receiver>/{id}
 
 For security reasons, many WebHook receivers require that the URI is an *https* URI and in some cases it must also contain an additional query parameter which is used to enforce that only the intended party can send WebHooks to the URI above.
 
-The <em><receiver></em> component is the name of the receiver, for example <em>github</em> or <em>slack</em>.
+The <em>&lt;receiver&gt;</em> component is the name of the receiver, for example <em>github</em> or <em>slack</em>.
```

Comment:
Let's go with the following syntax instead:

```
`<receiver>`
```

We avoid HTML elements whenever possible. Also, code-fencing this text eliminates the need to encode the symbols.